metricbeat/module/system/core/config.go

```diff
@@ -0,0 +1,46 @@
+package core
+
+import (
+	"strings"
+
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/pkg/errors"
+)
+
+// Core metric types.
+const (
+	percentages = "percentages"
+	ticks       = "ticks"
+)
+
+// Config for the system core metricset.
+type Config struct {
+	Metrics  []string `config:"core.metrics"`
+	CPUTicks *bool    `config:"cpu_ticks"` // Deprecated.
```

Comment:
Does it make sense to add deprecation warnings in 5.6 when there is no replacement for cpu_ticks in 5.6? I thought there should be some period of overlap where the two config options are available to aid in migration.